apps/content-gateway-api/src/app/authorization/authorization.ts

```diff
@@ -0,0 +1,204 @@
+    name: "Allows finding schemas for object having schema info",
+    operationName: FIND_SCHEMA_FOR,
+    policies: [allowAllPolicy()],
+});
+
+const allowRemoveSchema: Permission<SchemaEntity, void> = {
+    name: "Allows removing schemas",
+    operationName: REMOVE_SCHEMA,
+    policies: [allowAllPolicy()],
+};
+
+const allowCreateUser: Permission<CreateUserParams, ContentGatewayUser> = {
+    name: "Allows creating users",
+    operationName: CREATE_USER,
+    policies: [allowAllPolicy()],
+};
+
+const allowCreateApiKey: Permission<CreateAPIKeyParams, APIKey> = {
+    name: "Allows creating API Keys",
+    operationName: CREATE_API_KEY,
+    policies: [allowAllPolicy()],
+};
+
+const allowDeleteUser: Permission<string, void> = {
+    name: "Allows deleting users",
+    operationName: DELETE_USER,
+    policies: [allowAllPolicy()],
+};
+
+const allowDeleteApiKey: Permission<DeleteAPIKeyParams, void> = {
+    name: "Allows deleting API Keys",
+    operationName: DELETE_API_KEY,
+    policies: [allowAllPolicy()],
+};
+
+const allowRemoveSchemaForOwn: Permission<SchemaEntity, void> = {
+    name: "Allows removing schemas for the current user",
+    operationName: REMOVE_SCHEMA,
+    policies: [allowForSelfPolicy()],
+};
+
+const allowSaveData = (): Permission<SaveDataParams, void> => ({
+    name: "Allows saving data",
+    operationName: SAVE_DATA,
+    policies: [allowAllPolicy()],
+});
+
+const allowSaveDataForOwn = (): Permission<SaveDataParams, void> => ({
+    name: "Allows saving data for schemas that are owned by the current user",
+    operationName: SAVE_DATA,
+    policies: [allowForSelfPolicy()],
+});
+
+const allowCreateApiKeyForOwn: Permission<CreateAPIKeyParams, APIKey> = {
+    name: "Allows creating API Keys for the current user",
+    operationName: CREATE_API_KEY,
+    policies: [allowForSelfPolicy()],
+};
+
+const allowDeleteApiKeyForOwn: Permission<DeleteAPIKeyParams, void> = {
+    name: "Allows deleting API Keys for their owner",
+    operationName: DELETE_API_KEY,
+    policies: [allowForSelfPolicy()],
+};
+
+// * Anons can't do anything for now.
+const anonymousPermissions: AnyPermission[] = [];
+
+const userPermissions: AnyPermission[] = [
+    allowRegisterSchema,
+    allowFindSchemaFor<any>(),
+    allowRemoveSchemaForOwn,
+    allowSaveDataForOwn(),
+    allowCreateApiKeyForOwn,
+    allowDeleteApiKeyForOwn,
+];
+
+const adminPermissions: AnyPermission[] = [
+    allowRegisterSchema,
+    allowLoadSchemaStats,
+    allowFindSchemaFor<any>(),
+    allowRemoveSchema,
+    allowCreateUser,
+    allowCreateApiKey,
+    allowDeleteUser,
+    allowDeleteApiKey,
+    allowSaveData(),
+];
+
+const rootPermissions: AnyPermission[] = [
+    allowRegisterSchema,
+    allowLoadSchemaStats,
+    allowFindSchemaFor<any>(),
+    allowRemoveSchema,
+    allowCreateUser,
+    allowCreateApiKey,
+    allowDeleteUser,
+    allowDeleteApiKey,
+    allowSaveData(),
+];
+
+const ANON_USER_ID = "1fa0fd73-03cb-4b3c-8350-47a2d7c27dc4";
```

Comment:
None that I know of. Technically it is not a good practice to hard-code ids, but in this case this is for the anonymous role which is something that is assigned when you are not authorized (so it could be anything really).